### PR TITLE
Bump dependencies in cli crate

### PR DESCRIPTION
**Pull request by satyakommula96**
## Which issue does this PR close?

Closes #.

## What changes are included in this PR?

- Bump datafusion-cli dependencies
   - use stabilized version aws-sdk 1.x.x (https://www.infoworld.com/article/3711271/aws-delivers-aws-sdk-for-rust.html)
   - use latest clap version

## Are these changes tested?
yes

### CODE REVIEW

File: datafusion-cli/src/print_format.rs [hunk 1]
```diff
@@ -39,11 +39,24 @@ pub enum PrintFormat {
     Automatic,
 }
 
+impl std::fmt::Display for PrintFormat {
```
Comment (comphead) [inline datafusion-cli/src/print_format.rs:42]: is this change related to version bump?

Comment (satyakommula96) [reply datafusion-cli/src/print_format.rs]: @comphead removed

**Comment by alamb** [comment]
Thanks @satyakommula96  -- it seems like there are a few test failures on this PR. Can you please resolve them?
